services/core-web/src/components/Forms/incidents/IncidentCategorySelect.js

```diff
@@ -0,0 +1,129 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import { Form } from "@ant-design/compatible";
+import "@ant-design/compatible/assets/index.css";
+import { Typography, Alert, Checkbox, Row, Col } from "antd";
+import CustomPropTypes from "@/customPropTypes";
+
+const propTypes = {
+  disabled: PropTypes.bool,
+  data: CustomPropTypes.options,
+  meta: PropTypes.shape({
+    initial: PropTypes.array,
+    touched: PropTypes.bool,
+    error: PropTypes.string,
+  }).isRequired,
+  input: PropTypes.shape({
+    onChange: PropTypes.func.isRequired,
+    value: PropTypes.arrayOf(PropTypes.string),
+  }).isRequired,
+};
+
+const defaultProps = {
+  data: [],
+  disabled: false,
+};
+
+// an item is considered a child of another item when its subType matches the parent's value
+const IncidentCategorySelect = (props) => {
```

Comment:
Not major, but `IncidentCategorySelect` makes it sound like it's a dropdown select menu to me.  Maybe we could go with something more specific to checkboxes or more generic?

Reply:
IncidentCheckboxGroup? I agree, this is not a very good name!